Documentation/observability/metrics.rst

```diff
@@ -1552,17 +1556,20 @@ Name                                     Labels
 ``bootstrap_seconds``                    ``source_cluster``                           Duration in seconds to complete bootstrap
 ======================================== ============================================ ========================================================
 
-Remote clusters
-~~~~~~~~~~~~~~~
+Clustermesh
+~~~~~~~~~~~
 
-==================================== ======================================= =================================================================
-Name                                 Labels                                                       Description
-==================================== ======================================= =================================================================
-``remote_clusters``                  ``source_cluster``                      The total number of remote clusters meshed with the local cluster
-``remote_cluster_failures``          ``source_cluster``, ``target_cluster``  The total number of failures related to the remote cluster
-``remote_cluster_last_failure_ts``   ``source_cluster``, ``target_cluster``  The timestamp of the last failure of the remote cluster
-``remote_cluster_readiness_status``  ``source_cluster``, ``target_cluster``  The readiness status of the remote cluster
-==================================== ======================================= =================================================================
+Note that these metrics are not prefixed by ``clustermesh_``.
+
+=============================================== ============================================================ ==================================================================
+Name                                            Labels                                                       Description
+=============================================== ============================================================ ==================================================================
+``remote_clusters``                             ``source_cluster``, ``source_node_name``                     The total number of remote clusters meshed with the local cluster
+``remote_cluster_failures``                     ``source_cluster``, ``source_node_name``, ``target_cluster`` The total number of failures related to the remote cluster
+``remote_cluster_last_failure_ts``              ``source_cluster``, ``source_node_name``, ``target_cluster`` The timestamp of the last failure of the remote cluster
+``remote_cluster_readiness_status``             ``source_cluster``, ``source_node_name``, ``target_cluster`` The readiness status of the remote cluster
+``leader_election_master_status``               ``name``                                                     The leader election status
```

Comment:
Nit. The last one should be in a separate category, as not related to the clustermesh subsystem.